charts/zabbix/values.yaml

```diff
@@ -309,13 +309,13 @@ zabbixweb:
   #saml_certs_secret_name: zabbix-web-samlcerts
   service:
     # -- Type of service for Zabbix web
-    type: ClusterIP
+    type: NodePort
     # -- Cluster IP for Zabbix web
     clusterIP:
     # -- Port to expose service
     port: 80
     # -- NodePort port to allocate (only if service.type = NodePort)
-    #nodePort: 31080
+    nodePort: 31080
```

Comment:
Same as above regarding NodePort service

Reply:
Hi @sa-ChristianAnton!

To use ingress, the service must be NodePort. I switched to make it easier. But I understand your concerns and have rolled back.